samples/data-sealing/enclave_a_v1/Makefile

```diff
@@ -7,6 +7,7 @@ CFLAGS=$(shell pkg-config oeenclave-$(C_COMPILER) --cflags)
 CXXFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --cflags)
 LDFLAGS=$(shell pkg-config oeenclave-$(CXX_COMPILER) --libs)
 INCDIR=$(shell pkg-config oeenclave-$(C_COMPILER) --variable=includedir)
+CRYPTO_LDFLAGS=$(shell pkg-config oeenclave-mbedtls_crypto --libs)
```

Comment:
Same here (and the rest of the samples) even we only support Mbed TLS for now.

Reply:
I did not get the comment here. Do you want me to write it as oeenclave-${CRYPTO_LIB} and declare CRYPTO_LIB as a variable in high level Makefile.

Reply:
Yes. Just the way you're going to support samples with OpenSSL.